### ISSUE TITLE
Is the project under development?

### ISSUE THREAD

**Issue by brauliodiasribeiro** [danielmiessler/Substrate#26]
Hii, Daniel's projects are fantastic.

This one isn't running yet? Is it in development?

**Comment by danielmiessler** [comment]
Yes, still being developed. Slow and steady!

**Comment by danielmiessler** [comment]
Yes, sorry for the pause. We are going crazy with it now. Looking forward to contributions.
